docs/accounts/Basics.md

```diff
@@ -0,0 +1,39 @@
+---
+tags: [Accounts]
+---
+
+# Account Basics
+
+Accounts are an entity which hold information about the type product a client may wish to open, and validate that one or more parties can open a product of that type. Each account can have both multiple Parties and multiple Portfolios associated to it.
```

Comment:
`....may wish to open, and validate that one or more parties....`

This transition doesn't quite scan to me, is this saying that an Account "validates that one or more parties can open a product of that type" ? And does that translates in practical terms to kyc checks?

Reply:
Based on the assumptions in my previous comment, I ran it through:
https://hemingwayapp.com/
and got
```
Accounts hold information about the type of product a client may wish to open. It also validates that one or more parties can open a product of that type. Each account can have both many Parties and many Portfolios associated to it. 
```

Reply:
I'll try and reword it 👍🏻